spec/spec_helper.rb

```diff
@@ -50,6 +53,17 @@
 
   c.include_context :stubbed_logger
 
+  c.before(:suite) do
+    analytics_config = Tempfile.new('analytics.yml')
+    analytics_config.write(YAML.dump(disabled: true))
+    analytics_config.close
+    ENV['PDK_ANALYTICS_CONFIG'] = analytics_config.path
+  end
```

Comment:
Is there a case where this is better than `PDK_DISABLE_ANALYTICS=1`?

Reply:
Yeah, this is in place so that we can test the behaviour of the config modification (interview, fetching from bolt config, etc) without being effected by the user's real config.